Octokit.Reactive/Clients/IObservableOrganizationsClient.cs

```diff
@@ -16,6 +16,12 @@ public interface IObservableOrganizationsClient
         IObservableTeamsClient Team { get; }
 
         /// <summary>
+        /// A client for GitHub's Organization Hooks API.
+        /// </summary>
+        /// <remarks>See <a href="http://developer.github.com/v3/orgs/hooks/">Hooks API documentation</a> for more information.</remarks>
+        IObservableOrganizationHooksClient Hooks { get; }
```

Comment:
Yep, there has been a move towards ensuring all property names for clients are singular (#53 #1035 #1036 #375 #1038), 
so I agree these should be `IOrganizationHooksClient Hook { get; private set; }`

It also makes me realise we have quite a few other clients that still need to get resolved!